### DISCUSSION TITLE
Spelling checks

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#160] [Q&A]
The skill system is coming along nicely, so I figured I'd spend some time fixing up some spellings on documentation and variable names here and there if that's alright with you?

**Comment by MokhaLeee** [comment]
spell error fix is okay, contribution on doc is encouraged, Large-scale code modification will be treated with caution.

**Comment by MokhaLeee** [comment]
also not encouraged to modify code style be cause fireblem8u's code style itself is chaotic and failed to strictly follow the code style of fe6 (although contributors have reached an agreement on that)
